chapters/emergencystop.adoc

```diff
@@ -0,0 +1,29 @@
+== Emergency stop
+
+This rule only applies to Division A.
+
+.Definition
+
+A team can ask to stop the game immediately after a grace period of 10 seconds or at the next stoppage, whichever happens first regardless of the current situation. It will receive a yellow card for this and must take a timeout immediately. If the team is out of timeouts, it is still allowed to remove robots from the field, but can not use any remaining timeout time.
+
+There are three possible situations when this rule is used:
```

Comment:
This is misleading. The rule can be used "whenever". These three options are results that could happen if the rule is used and how to proceed with the game afterwards

Reply:
I see, I'll re phrase it